docker/README.md

```diff
@@ -1,56 +1,60 @@
 
-Second, initialize a new UserFrosting project:
+You may be tempted to run with this in production but this setup has not been security-hardened. For example:
 
-1. Copy `app/sprinkles.example.json` to `app/sprinkles.json`
-2. Run `chmod 777 app/{logs,cache,sessions}` to fix file permissions for web server. (NOTE: File
-   permissions should be properly secured in a production environment!)
-3. Run `docker-compose run composer install --ignore-platform-reqs --no-scripts` to install all composer modules. (https://hub.docker.com/_/composer) Sometimes dependencies or Composer scripts require the availability of certain PHP extensions. You can work around this as follows: Pass the `--ignore-platform-reqs and --no-scripts` flags to install or update
-4. Run `docker-compose run node npm install` to install all npm modules.
-5. Run `docker-compose run composer update --ignore-platform-reqs --no-scripts` to install remaining composer modules
-6. Run `docker-compose run node npm run uf-assets-install` to install all frontend vendor assets.
+- Database is exposed on port 8593 so you can access MySQL using your favorite client at localhost:8593. However,
+  the way Docker exposes this actually bypasses common firewalls like `ufw` so this should not be exposed in production.
+- Database credentials are hard-coded so obviously not secure.
+- File permissions may be more open than necessary.
+- HTTPS not implemented fully
+- It just hasn't been thoroughly tested in the capacity of being a production system.
 
-Now you can start up the entire Nginx + PHP + MySQL stack using docker with:
 
-    $ docker-compose up -d
+## Environment Dependencies
 
-the `-d` flag will launch this in the background so you can continue to use the terminal window. On the first run you need to init the database (your container name may be different depending on the name of your root directory):
+Though Docker will take care of this for the most part, there are 2 things you'll need to get started.
 
-    $ docker exec -it -u www-data userfrosting_php_1 sh -c 'php bakery migrate'
+* [Docker Compose](https://docs.docker.com/compose/install/)
+* Bash plus the common Linux tools (for Windows users, Git Bash will cover this)
 
-You also need to setup the first admin user (again, your container name may be different depending on the name of your root directory):
+## First Run
 
-    $ docker exec -it -u www-data userfrosting_php_1 sh -c 'php bakery create-admin'
+>Certain `bakery` commands such as `bake` that use NodeJS are not currently supported but should work provided a suitable image is used.
 
-Now visit `http://localhost:8591/` to see your UserFrosting homepage!
+Run the following through `bash` from the project root directory;
 
-**Paste these into a bash file and execute it!**
+```bash
+# Create ".env" and "sprinkles.json" from examples
+cp app/sprinkles.example.json app/sprinkles.json
+cp app/.env.example app/.env # This will need to be updated to reflect docker-compose.yml
```

Comment:
By the user, but it turns out it isn't needed anyway because the true environment variables that are hard coded into the Docker configuration override it. Updated to reflect this.